MarsRover.Test/RoverTest.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using Xunit;
+
+namespace MarsRover.Test
+{
+    public class RoverTest
+    {
+        
+        [Theory]
+        [InlineData("(0,1,e)", "MML", "(2,1,N)")]
+        [InlineData("(0,0,N)", "MmM", "(0,3,N)")]
+        [InlineData("(10,5,n)", "MMLMrmML", "(9,9,W)")]
+        public void rover_test(string position, string moves, string expected)
```

Comment:
The name of the method should be proper.
Please refer to how to write method names in test cases.

Reply:
resolved